ZuluContent/Zulu/Items/SingleClick/SingleClickHandler.cs

```diff
@@ -32,17 +32,24 @@ private static (IEnumerable<string>, IEnumerable<string>) GetAffixes(IMagicItem
             return (prefixes, suffixes);
         }
 
-        private static void DefaultHandleSingleClick<T>(T item, Mobile m) where T : Item, IMagicItem
+        public static string GetMagicItemName(IMagicItem item)
```

Comment:
This was more for POC than final implementation, I'm thinking we should refactor these methods out into a seperate module rather

Reply:
What kind of module are you thinking of? 

In C# static classes with static functions let you import them into other classes with the [`using static` directive](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-static) which is about as close as you get to just having a file with functions in it you can reuse.

`GetMagicItemName` is definitely useful elsewhere for sure :+1:

Reply:
ah sweet thanks for the heads up on that, will use that then :)